### ISSUE TITLE
Outside of Docker: Download failed with status 500 Internal Server Error

### ISSUE THREAD

**Issue by CpBobette** [unifi-toolbox/unifi-protect-video-downloader#41]
I want to run the tool as a script and not within docker.

I did checkout the commit marked for the dockerhub release and built the .whl using poetry and installed it in a venv, then I tried running the script but I keep getting this error.
The data exists on the CK because when I run with docker I download the file when using the same arguments.

Did I miss anything in the ''build'' process ?

I am running on Windows.

```
Getting camera list
Successfully authenticated as user admin
Successfully retrieved data from /api/cameras
Cameras found:
- Camera 1 (601847cd0320bd03e70003f6)
Downloading video files between 2021-02-01 18:00:00 and 2021-02-02 00:00:00 from 'https://192.168.10.234:7443/api/video/export' for camera Camera 1

Downloading footage for camera 'Camera 1' (601847cd0320bd03e70003f6)
Downloading video for time range 2021-02-01 18:00:00 - 2021-02-01 18:59:59 to D:\Video/2021/02/01/Camera 1 (03f6)/Camera 1 (03f6) - 2021-02-01 - 18.00.00.mp4
Download failed with status 500 Internal Server Error
Traceback (most recent call last):
  File "d:\unifitool\testenv\lib\site-packages\protect_archiver\client.py", line 171, in download_file
    raise cls(
protect_archiver.client.DownloadFailed: Download failed with status 500 Internal Server Error:        
No files found matching criteria.
Retrying in 3 second(s)...
```

**Comment by CpBobette** [comment]
The timestamps in the video download GET requests are not same when running in docker vs diretly in python.


```
 "GET /api/video/export?&camera=601847cd0320bd03e70003f6&start=1612220400000&end=1612223999000
 "GET /api/video/export?&camera=601847cd0320bd03e70003f6&start=1612202400000&end=1612205999000
```

**Comment by danielfernau** [comment]
Hi @CpBobette 

An error 500 indicates that something went wrong internally on the CloudKey, in most cases because the system was unable to handle the request within the Protect software. A common reason for this would be that you're trying to download footage from a specific camera when there are no recordings available for the requested time frame (see also #26).

As for the differences between the timestamps, keep in mind that, among other things, different Python- and module versions can affect how the code works internally. Make sure you're using [Python 3.8.2](https://www.python.org/downloads/release/python-382/) in combination with the module versions as listed in the [pyproject.toml](https://github.com/unifi-toolbox/unifi-protect-video-downloader/blob/master/pyproject.toml) and [poetry.lock](https://github.com/unifi-toolbox/unifi-protect-video-downloader/blob/master/poetry.lock) files. The easiest way to make sure that both the Python version as well as the project's dependencies are correct and compatible is to use [Python Poetry](https://python-poetry.org/docs/).

In case you need additional help, please include the full command line you're using to run the script (remove/redact username/password/personal information).

Hope this helps!
